recipes/frv/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -ex
+
+mkdir -pv build
+pushd build
+
+cmake \
+	-DCMAKE_BUILD_TYPE:STRING=Release \
+	-DCMAKE_INSTALL_LIBDIR:PATH="lib" \
+	-DCMAKE_INSTALL_PREFIX:PATH=${PREFIX} \
+	${SRC_DIR}
+
+# build
+cmake --build . --parallel ${CPU_COUNT} --verbose
+
+# install
+cmake --build . --parallel ${CPU_COUNT} --verbose --target install
```

Comment:
Is there a `ctest` or make check/test we ca run here?

Reply:
@ocefpaf, not yet, but we can lean on the developers to add a suite.